web/portal/platform/src/translations/ca.json

```diff
@@ -1,5 +1,6 @@
 {
   "'https://' URLs valid only (without trailing '/')": "Només s'accepten adreces 'https://' (sense '/' al final)",
+  "Accept": "AcceptarWhite",
```

Comment:
AcceptarWhite?

Reply:
You have included this platform portal change in a commit tagged as portal/brand